docs/introduction.rst

```diff
@@ -53,128 +56,192 @@ Survey of North America (BBS) and the Alwyn Gentry Tree Transect data:
-          ls                  display a list all available dataset scripts
-          citation            view citation
-          reset               reset retriever: removes configation settings,
-                              scripts, and cached data
-          help
-
-      optional arguments:
-        -h, --help            show this help message and exit
-        -v, --version         show program's version number and exit
-        -q, --quiet           suppress command-line output
-
-
-To install datasets, use ``retriever install``::
+      {download,install,update,new,new_json,edit_json,delete_json,ls,citation,reset,help}
+                            sub-command help
+        download            download raw data files for a dataset
+        install             download and install dataset
+        update              download updated versions of scripts
+        new                 create a new sample retriever script
+        new_json            CLI to create retriever datapackage.json script
+        edit_json           CLI to edit retriever datapackage.json script
+        delete_json         CLI to remove retriever datapackage.json script
+        ls                  display a list all available dataset scripts
+        citation            view citation
+        reset               reset retriever: removes configation settings,
+                            scripts, and cached data
+        help
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -v, --version         show program's version number and exit
+      -q, --quiet           suppress command-line output
+
+To install datasets, use the ``install`` command.
+
+Examples
+~~~~~~~~
+
+**Using install**
+
+The install command downloads the datasets and installs them in the desired engine.
+
+$ ``retriever install -h`` (gives install options)
+
+::
 
     usage: retriever install [-h] [--compile] [--debug]
-                             {mysql,postgres,sqlite,msaccess,csv} ...
+                             {mysql,postgres,sqlite,msaccess,csv,json,xml} ...
 
     positional arguments:
-      {mysql,postgres,sqlite,msaccess,csv}
+      {mysql,postgres,sqlite,msaccess,csv,json,xml}
                             engine-specific help
         mysql               MySQL
         postgres            PostgreSQL
         sqlite              SQLite
         msaccess            Microsoft Access
         csv                 CSV
+        json                JSON
+        xml                 XML
 
     optional arguments:
       -h, --help            show this help message and exit
       --compile             force re-compile of script before downloading
       --debug               run in debug mode
 
 
-Examples
-~~~~~~~~
+**Examples using install with engines**
+
+
+These examples are using Breeding Bird Survey data (breed-bird-survey)
+The retriever has support for various engines; mysql, postgres, sqlite, msaccess, csv, json, xml, download_only.
```

Comment:
Most people won't know what we mean by "engines" so lets say "...for various databases and flat file formats (mysql, postgres, sqlite, msaccess, csv, json, xml)".